Sprint-3/2-practice-tdd/count.js

```diff
@@ -1,5 +1,21 @@
 function countChar(stringOfCharacters, findCharacter) {
-  return 5
+  if (
+    typeof stringOfCharacters !== "string" ||
```

Comment:
If either or both arguments are not strings, does it really make sense to return 0? Think of it this way, you call the `countChar(2, 3)`, what would you expect your function to return? I would suggest you to check this link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error